From e15216f8424432ce5314bb7c84a7e8e3cc2f6f0d Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 20:22:08 +0900
Subject: [PATCH 526/909] arm64: dts: r8a7795: Add cpg reset for LVDS Interface

commit 328e2573c973491fa8432f5038a5ed90fc8bd9a2 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

It is necessary to reset the LVDS Interface according to display on/off.
Therefore, this patch adds CPG reset properties in DU device node
or the R8A7795 SoC.

Reported-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
[takeshi.kihara.df: rebased]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 629f5659422a..ffea3667ca2c 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -3064,6 +3064,11 @@
 				 <&cpg CPG_MOD 722>,
 				 <&cpg CPG_MOD 721>;
 			clock-names = "du.0", "du.1", "du.2", "du.3";
+			resets = <&cpg 724>,
+				 <&cpg 724>,
+				 <&cpg 722>,
+				 <&cpg 722>;
+			reset-names = "du.0", "du.1", "du.2", "du.3";
 			vsps = <&vspd0 0 &vspd1 0 &vspd2 0 &vspd0 1>;
 			status = "disabled";
 
-- 
2.17.1

